### PR TITLE
Sprint2

### PR DESCRIPTION
**Pull request by cbizon**

### CODE REVIEW

File: PrioritiesSprint2.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- finalize mvp1-template1-clinical-cps; push to CI at end of sprint
+- finalize mvp1-template3-openpredict; push to CI at end of sprint
+
+# Fix Failing Automated Tests
+## Octopus (CI):
+- Resolve Test-running Issues
+- Each ARAs should return no more than 75 failures
+- Each ARAs should have no more than 75 tests in which they do not return results
+- CQS should return no more than 75 failures
+- Implement all test runs from the Test Run Schedule that can currently be supported
```
Comment (maximusunc) [inline PrioritiesSprint2.md:73]: I had added a couple more items to the spreadsheet:
- Implement MVP2 tests
- Coordinate with ARAs to facilitate completion of work.

Comment (cbizon) [reply PrioritiesSprint2.md:73]: Is Implement MVP2 tests a replacement for "Implement all test runs from the Test Run Schedule..."? 

Who is doing the coordinating?  The Testing WG?

Comment (maximusunc) [reply PrioritiesSprint2.md:73]: No, the Test Run Schedule doesn't include a differentiation between MVP1 and MVP2 tests. The Testing WG is coordinating, yes.

Comment (cbizon) [reply PrioritiesSprint2.md:73]: Gotcha, I've updated the PR - thanks

File: PrioritiesSprint2.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+## Octopus (CI):
+- Sending Concurrent queries (MVP1&MVP2) in batch of 5,10,15,20,25,30 (CI Env)
+## Lobster (Test):
+- Sending Concurrent queries (MVP1&MVP2) in batch of 5,10,15,20,25,30 (TEST Env)
+
+# CQS Workflows
+## Pre-development
+- Develop new MVP1 and MVP2 workflows
+## Eel (DEV):
+- develop mvp1-template4-infores:aeolus; push to CI at end of sprint
```
Comment (karafecho) [inline PrioritiesSprint2.md]: Correction: mvp1-template4-service-provider-aeolus. Add: mvp1-template5-spoke-chembl.

File: PrioritiesSprint2.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+# Develop Pathfinder Query
+## Pre-development work:
+- Map out upcoming cycles culminating in release to production
+- Initial UI planning and discussions
+- Document the use of TRAPI for Pathfinder queries in the ReasonerAPI github repo.
+- Provide input variables and data examples to the UI team
+## Octopus (CI) work:
+- ARAs implement prototype Pathfinder
+- Evaluate performance and result quality
```
Comment (andrewsu) [inline PrioritiesSprint2.md:13]: Are there more details on how this evaluation will be done?

File: PrioritiesSprint2.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+- DM and UI collaborate to ensure that the provenance of new inferred treats edges is correctly represented (design).
+## Octopus (CI):
+- ARAs implement new reasoning and scoring based on changes in treats edges from KPs
+- Implement new CQS templates to regenerate former treats edges as predictions with aux graphs referring to the foundational edge.
+
+# New Merged Scores
+## Pre-Development
+- Implement systematic analysis of results
+- Update scoring document to include ARA scores
+- Add DCG metrics to O&O benchmarks
```
Comment (andrewsu) [inline PrioritiesSprint2.md:43]: Can we clarify what the "O&O benchmarks" are? For example, can we point to other metrics that are already implemented next to which DCG would be implemented?

File: PrioritiesSprint2.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+## Eel (DEV):
+- develop mvp1-template4-infores:aeolus; push to CI at end of sprint
+## Octopus (CI):
+- finalize mvp1-template1-clinical-cps; push to CI at end of sprint
+- finalize mvp1-template3-openpredict; push to CI at end of sprint
+
+# Fix Failing Automated Tests
+## Octopus (CI):
+- Resolve Test-running Issues
+- Each ARAs should return no more than 75 failures
```
Comment (gglusman) [inline PrioritiesSprint2.md]: lines 67/68: "Each ARAs" -> "Each ARA"

File: PrioritiesSprint2.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+## Lobster (Test):
+- Sending Concurrent queries (MVP1&MVP2) in batch of 5,10,15,20,25,30 (TEST Env)
+
+# CQS Workflows
+## Pre-development
+- Develop new MVP1 and MVP2 workflows
+## Eel (DEV):
+- develop mvp1-template4-infores:aeolus; push to CI at end of sprint
+## Octopus (CI):
+- finalize mvp1-template1-clinical-cps; push to CI at end of sprint
```
Comment (karafecho) [inline PrioritiesSprint2.md]: Correction: mvp1-template1-clinical-kps; push to TEST at end of sprint. Add: Information Radiator and TAQA testing.

File: PrioritiesSprint2.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+- finalize mvp1-template3-openpredict; push to CI at end of sprint
```
Comment (karafecho) [inline PrioritiesSprint2.md]: Correction: push to TEST at end of sprint. Add: Information Radiator and TAQA testing.

**Review by karafecho — COMMENTED** [review]
I added a few corrections and additions.

**Review by vdancik — APPROVED** [review]

**Review by sierra-moxon — APPROVED** [review]

File: PrioritiesSprint2.md (hunk 3 above)
Comment (cbizon) [reply PrioritiesSprint2.md:13]: @uhbrar can you comment?

File: PrioritiesSprint2.md (hunk 4 above)
Comment (cbizon) [reply PrioritiesSprint2.md:43]: Based on the O&O call yesterday, I'd say that this is not completely resolved.  There is a colab framework that Ewy has been using to put together some different benchmarks like calculating number of ties.  But there is also the benchmarking work coming out of the Test WG.    There is some hope that we can merge these two together, but the shape of that is not completely clear at this point.

File: PrioritiesSprint2.md (hunk 5 above)
Comment (cbizon) [reply PrioritiesSprint2.md]: Thanks

**Comment by cbizon** [comment]
@karafecho I think I addressed your corrections, let me know if not

**Comment by karafecho** [comment]
Yes, although the typo was not corrected: mvp1-template1-clinical-kps (not "cps").

**Comment by cbizon** [comment]
Thanks @karafecho , typo fixed now.

**Review by CaseyTa — APPROVED** [review]

**Pull request merged**